src/libs/onyxSubscribe.ts

```diff
@@ -0,0 +1,15 @@
+import Onyx, {ConnectOptions} from 'react-native-onyx';
+import {OnyxKey} from 'react-native-onyx/lib/types';
```

Comment:
```suggestion
import {OnyxKey} from '../ONYXKEYS';
```

You can import `OnyxKey` from our own definitions 😄

Reply:
Done!